_posts/2019-03-07-strimzi-0.11.0-release.md

```diff
@@ -0,0 +1,191 @@
+Now, you can customize the advertised hostname and port.
+Strimzi will then automatically configure the advertised address in the Kafka brokers and add it to the broker certificates so it can be used for TLS hostname verification.
+
+Here's an example snippet for overriding advertised addresses.
+
+```yaml
+apiVersion: kafka.strimzi.io/v1alpha1
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    ...
+    listeners:
+      external:
+        type: route
+        authentication:
+          type: tls
+        overrides:
+          brokers:
+          - broker: 0
+            advertisedHost: example.hostname.0
+            advertisedPort: 12340
+          - broker: 1
+            advertisedHost: example.hostname.1
+            advertisedPort: 12341
+          - broker: 2
+            advertisedHost: example.hostname.2
+            advertisedPort: 12342
+```
+
+You can find many more overrides option in the official documentation in the Kafka broker listeners [chapter](https://strimzi.io/docs/latest/#assembly-configuring-kafka-broker-listeners-deployment-configuration-kafka).
+
+# Prometheus alerts
+
+Other than providing the server component for scraping metrics from your applications, the Prometheus project also provides an alerting system through the alert manager component.
+It is possible to declare alerting rules on the Prometheus server in order to be notified about specific conditions in the metrics.
+When an alert condition is evaluated as true, Prometheus sends alert data to the alert manager which then sends notifications out.
+Notifications can be sent via methods such as email, Slack, PagerDuty and HipChat.
+
+The new Strimzi release provides the Kubernetes/OpenShift resources for deploying the alert manager as well as few examples of alerting rules for Kafka and Zookeeper metrics and related notifications via Slack.
+
+Here's an example snippet with an alert related to the under replicated partitions metric.
+
+```yaml
+- alert: UnderReplicatedPartitions
+  expr: kafka_server_replicamanager_underreplicatedpartitions > 0
+  for: 10s
+  labels:
+    severity: warning
+  annotations:
+    summary: 'Kafka under replicated partitions'
+    description: 'There are {{ $value }} under replicated partitions on {{ $labels.kubernetes_pod_name }}'
+```
+
+# ... and many more
+
+Other features were included in this new release, the most important ones:
+
+* Support for the Cluster Operator to watch all namespaces for the supported resources.
+* Allow users to configure the default ImagePullPolicy related to the containers images download.
+* Operator Lifecycle Manager integration which allowed to have Strimzi as part of the [OperatorHub.io](https://www.operatorhub.io/). You can get more information on this [post](https://strimzi.io/2019/03/06/strimzi-and-operator-lifecycle-manager.html)
+
+Of course, bug fixes are there as well!
+
+# Conclusion
+
+This release represents another really huge milestone for this open source project.
```

Comment:
Just thinking ... is it really _a huge milestone_? Wouldn't _another milestone_ be enough? Feel free to ignore this if you think its really a huge milestone :-D

```suggestion
This release represents another milestone for this open source project.
```